src/metpy/calc/thermo.py

```diff
@@ -2552,7 +2931,7 @@ def vertical_velocity(omega, pressure, temperature, mixing_ratio=0):
 
 
 @exporter.export
-@preprocess_xarray
+@preprocess_and_wrap(wrap_like='dewpoint', broadcast=('dewpoint', 'pressure'))
```

Comment:
The order does matter in that it controls the output dimension ordering, since the earlier arguments have their dimensions come first. For example, if dewpoint has dims (time, isobaric, y, x) and pressure is (isobaric), then dewpoint, pressure ordering will make the broadcasted result (time, isobaric, y, x) [what we want], whereas pressure, dewpoint ordering would give (isobaric, time, y, x) [bad]. Because of this, whenever arguments that are likely to be coordinates occur, I've tried to put those at the end of the list in order to minimize unexpected transposition.